### ISSUE TITLE
server reload can not work on windows

### ISSUE THREAD

**Issue by lianyzhou** [glenjamin/ultimate-hot-reloading-example#13]
server side reloading works great on ubuntu , but not work in **windows**.

when after npm install , npm start , and visit localhost:3000 , the browser report babel-runtime error,

when you npm install babel-runtime , the error disappear

**Comment by glenjamin** [comment]
Oh, good spot - Babel-runtime is missing. I'll get that added.

**Comment by glenjamin** [comment]
So I looked into this, and when I do `npm install` babel runtime gets included.

Could you give me the output of `npm version` please?

**Comment by lianyzhou** [comment]
C:\Users\lianyi>npm version
{ npm: '3.3.12',
  ares: '1.10.1-DEV',
  http_parser: '2.6.0',
  icu: '56.1',
  modules: '47',
  node: '5.5.0',
  openssl: '1.0.2e',
  uv: '1.8.0',
  v8: '4.6.85.31',
  zlib: '1.2.8' }

**Comment by lianyzhou** [comment]
when fix babel-runtime error , i also found that , server reload not work as desired in windows.

after spending some time in debugging , i found that , the problem as below.

watcher.on('ready', function() {
  watcher.on('all', function() {
    console.log("Clearing /server/ module cache from server");

```
Object.keys(require.cache).forEach(function(id) {
 **if (/(\/|\\)server(\/|\\)/.test(id)) delete require.cache[id];**
});
```

  });
});

the slash in windows is \ , but in linux is / , so after changing the regex , the problem resolves.

hope to improve the  hot reloading example , Thanks

**Comment by glenjamin** [comment]
Good find! I'll get this updated soon (or will merge a PR 😀)

**Comment by glenjamin** [comment]
Fixed in d57bc1e
